### ISSUE TITLE
G2FM v7.3.0 update no longer boots my computer when I use UEFI

### ISSUE THREAD

**Issue by KaMyKaSii** [a1ive/grub2-filemanager#236]
BIOS still works. I am loading G2FM through Ventoy, using the iso multiarch or an entry in grub.cfg. G2FM v7.2.0 works perfectly in the same configuration. The screen stays frozen with an "_" in the upper left corner. Is there any way to provide relevant logs?

**Comment by a1ive** [comment]
Have you tried to load grubfmx64.efi instead of grubfm iso?

**Comment by KaMyKaSii** [comment]
> Have you tried to load grubfmx64.efi instead of grubfm iso?

Yep, it's in the grub.cfg entry, same result
```
if [ "${grub_platform}" == "pc" ]; then
	menuentry "Grub2 File Manager - Legacy BIOS" {
		set _path="/grub"
		linux ${_path}/loadfm
		initrd ${_path}/grubfm.iso
		}
else 
    menuentry "Grub2 File Manager - UEFI" {
        chainloader /efi/boot/grubfmx64.efi
	}
fi
```

**Comment by a1ive** [comment]
Do you use MacBook? Maybe the UEFI screenshot module is not compatible with you firmware.
Try to remove `crscreenshot` in module list (https://github.com/a1ive/grub2-filemanager/blob/master/arch/x64/builtin.lst)
[grub2-filemanager.tar.gz](https://github.com/a1ive/grub2-filemanager/files/4803386/grub2-filemanager.tar.gz)

**Comment by KaMyKaSii** [comment]
> Do you use MacBook? Maybe the UEFI screenshot module is not compatible with you firmware.
> Try to remove `crscreenshot` in module list (https://github.com/a1ive/grub2-filemanager/blob/master/arch/x64/builtin.lst)
> [grub2-filemanager.tar.gz](https://github.com/a1ive/grub2-filemanager/files/4803386/grub2-filemanager.tar.gz)

No, it's an old notebook. I tried the ISO that you provided in the tar.xz but the same problem. But "luckily" I found that the problem is the same as I am having when trying to boot the Linux Mint 20 beta iso using Ventoy in UEFI mode: everything is working, but the screen that should be GRUB2 is just a black screen. I was able to confirm this using G2FM v7.3.0 in BIOS mode to be able to see my partition and folder structure and then replicate the movements when using in UEFI mode (1 down, 1 enter, 14 down, 1 enter, 10 down, 1 enter, 2 down, 2 enter) then the selected ISO loaded normally. Do you have a hint of what it might be?

**Comment by a1ive** [comment]
Press F4 then press T to enter text mode.

**Comment by KaMyKaSii** [comment]
> Press F4 then press T to enter text mode.

It works! And after opening GRUB2 in text mode, pressing F4 and changing the resolution will make GRUB2 in graphic mode work again

**Comment by a1ive** [comment]
I checked the source code, and no code related to video display has been modified since v7.2.

**Comment by KaMyKaSii** [comment]
> I checked the source code, and no code related to video display has been modified since v7.2.
> Have you tried to boot grubfmx64.efi directly (not through Ventoy)?

Could you explain to me how to do this?

**Comment by a1ive** [comment]
Backup Ventoy's loader (/EFI/BOOT/BOOTX64.EFI) and rename grubfmx64.efi to BOOTX64.EFI to replace it.
